cogs/TFT.py

```diff
@@ -342,28 +344,28 @@ def get_recentmatch_embed(self, match_data, summoner, queue):
             title=f"Most recent match for {summoner}.",
         )
 
-        if placement == 1:
+        if api_placement or formatted_placement == 1:
```

Comment:
This returns true if api_placement exists. You meant to write this as 
` if (api_placement == 1) or (formatted_placement == 1):`